src/app/grid/grid-paste/grid-paste-sample.component.ts

```diff
@@ -126,14 +126,17 @@ export class GridPasteSampleComponent {
         for (const pkVal of updatedRecsPK) {
             const row = this.grid1.getRowByKey(pkVal);
             if (row) {
-            row.nativeElement.style["font-style"] = "italic";
-            row.nativeElement.style.color = "gray";
+                row.nativeElement.style["font-style"] = "italic";
```

Comment:
Huh, just noticed this bit... I don't get it, the transactions integration with the grid already applies "modified" styles, this shouldn't be needed at all. Especially, since modifying the row native elements is usually not a good idea with virtualization. Can you check if this is needed and remove it if not?